slackHook.d.ts

```diff
@@ -1,4 +1,4 @@
-import * as Transport from "winston-transport";
+import as Transport from "winston-transport";
```

Comment:
it looks like this line is not that right.

```suggestion
import Transport = require("winston-transport");
```